src/pages/exercise.js

```diff
@@ -0,0 +1,18 @@
+import Head from 'next/head';
+
+import ExerciseForm from '../components/ExerciseForm';
+import StyledFormContainer from '../components/StyledFormContainer';
+
+export default function ExerciseFormPage() {
```

Comment:
I would keep the name here the same as the file name. Just keep in mind that the file name should be written in kebab-case and the component name in PascalCase, e.g. `exercise-form.js` and `ExerciseForm`